client/keys/list.go

```diff
@@ -38,13 +37,13 @@ func runListCmd(cmd *cobra.Command, args []string) error {
 func QueryKeysRequestHandler(w http.ResponseWriter, r *http.Request) {
 	kb, err := GetKeyBase()
```

Comment:
Add content type JSON header:
`w.Header().Set("Content-Type", "application/json")`

same with the other REST handlers

Reply:
Good idea. I will change the code soon. Could you help to answer this [question](https://github.com/cosmos/cosmos-sdk/issues/2331)? It is about the circleci test failure of this PR.